### DISCUSSION TITLE
Annot link

### DISCUSSION THREAD

**Discussion by Mano2731995** [pymupdf/PyMuPDF#2216] [Looking for help]
Hai jorj

What is the purpose of PDF_ANNOT_LINK? 

For my better understanding if you have any sample code pls share me Or explain me that annot![IMG_20230207_163858.jpg](https://user-images.githubusercontent.com/117537492/217228807-979e446b-efff-459c-b112-67b2c0287743.jpg)

**Comment by JorjMcKie** [comment]
In PDF, annotations, links and fields are technically the same - although the three categories are treated very differently.
Every PDF page having at least of the above, has an `/Annots` key in its definition (an array of xref numbers) which is not empty.
Every object definition behind such an annotation xref contains PDF keys `/Type/Annot /Subtype/xxx`. The definitions in PyMuPDF `fitz.PDF_ANNOT_xxx` represent this subtype xxx.
